src/state/state.go

```diff
@@ -24,9 +24,7 @@ type State struct {
 	paths        *PathsState
 	localManager *local.Manager
 	newPersisted []model.ObjectState
-	branches     map[string]*model.BranchState
-	configs      map[string]*model.ConfigState
-	configRows   map[string]*model.ConfigRowState
+	objects      *orderedmap.OrderedMap
```

Comment:
Ako sa pridali metody do interface `ObjectState`, tak to nema zmysel mat 3x, ... aj tak sa to vacsinou potom spajalo v `All()`.